Adyen App: Information about 15s timeout for Adyen calls (#1063)
saleor/saleor-docs: docs/developer/app-store/apps/adyen.mdx

Adyen App follows the flow described in detail in the [Saleor Payment App documentation](../../payments#saleor-payment-app).

## Limitations

This section contains known limitations of this App.

### Maximum timeout for Adyen calls is 15 seconds

> Saleor synchronous webhooks have a maximum response time limit of 20 seconds. To allow graceful error handling, app restricts Adyen response time to 15 seconds.

Reviewer: ```suggestion
Saleor synchronous webhooks have a maximum response time limit of 20 seconds. The app restricts Adyen response time to 15 seconds to allow graceful error handling.
```

> If Adyen surpass this limit, the App will return FAILURE status accompanied by an appropriate error message (Timeout Error).

Reviewer: ```suggestion
If Adyen surpasses this limit, the App will return a FAILURE status with an appropriate error message (Timeout Error).
```
